spec/controllers/queue_items_controller_spec.rb

```diff
@@ -3,200 +3,173 @@
 describe QueueItemsController do
   describe 'GET index' do
     it 'sets @queue_items with authentication' do
-      user = Fabricate(:user)
-      session[:user_id] = user.id
-      queue_items = user.queue_items
+      set_current_user
+      queue_items = current_user.queue_items
```

Comment:
queue_items is a local variable, and you can go further and define it with `let(:queue_items) { queue_items = current_user.queue_items }`